src/main/scala/org/antipathy/scoozie/builder/MonadBuilder.scala

```diff
@@ -0,0 +1,37 @@
+package org.antipathy.scoozie.builder
+
+/**
+  * Class for accessing values contained in Monads
+  */
+private[scoozie] object MonadBuilder {
+  import scala.util.{Failure, Success, Try}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/app/SimonJPegg/scoozie/pullRequest?prid=3072461)